FirebaseAuth/Tests/SampleSwift/AuthenticationExample/ViewControllers/OtherAuthMethodControllers/PasswordlessViewController.swift

```diff
@@ -31,8 +31,10 @@ class PasswordlessViewController: OtherAuthViewController {
 
   // MARK: - Firebase 🔥
 
-  private let authorizedDomain: String = "ENTER AUTHORIZED DOMAIN"
-  private let customDomain: String = "ENTER AUTHORIZED HOSTING DOMAIN"
+  private let authorizedDomain: String =
+    "fir-ios-auth-sample.firebaseapp.com" // Enter AUTHORIZED_DOMAIN
```

Comment:
@paulb777  is there a way to read these values somewhere hidden?

Reply:
I'll send you the link